README.md

```diff
@@ -34,8 +37,9 @@ var add = function (x, y) {
   return x + y
 }
 ```
-
--- YOUR ANSWER HERE --
+es5
+yes....? because variable add references the anonymous function but the fuction will be asocciated with variable add
+add(x,y)
```

Comment:
Again, we would prefer that you write `add(3,99.5);` or something similar to make it more concrete. But this is still an acceptable answer